SETUP.md

```diff
@@ -9,6 +9,8 @@ Setup is supported on Linux and macOS, but [**not on Windows**](https://github.c
 
 First, you must configure two environment variables (`OWNER` and `REPO`).
 These variables specify the GitHub repository for the manuscript (i.e. `https://github.com/OWNER/REPO`).
+Make sure that the case of `OWNER` matches how your account name is displayed on GitHub.
```

Comment:
Should we say username rather than "account name" which is how GitHub docs say it

Reply:
I used account name to match the lines below.  I'll change it in both places.